src/main/resources/default/taglib/t/blobImageSoftRefField.html.pasta

```diff
@@ -130,5 +147,37 @@
             }
             return '/' + path.substring(0, path.lastIndexOf('/') + 1);
         }
+
+        function isImage(extension) {
+            return extension === '.jpg' || extension === '.jpeg' || extension === '.png';
+        }
+
+        function determineFileIcon(extension) {
+            if (extension === '.pdf') {
+                return 'fa-file-pdf';
+            }
+
+            if (extension === '.tif' || extension === '.tiff' || extension === '.bmp' || extension === '.svg') {
```

Comment:
png? can't use isImage() here?

Reply:
PNG files will be displayed as regular images and thus do not require an icon.